src/boot/efi/meson.build

```diff
@@ -266,11 +266,19 @@ efi_ldflags = [
         efi_crt0,
 ]
 
-possible_link_flags = [
-        '-Wl,--no-warn-execstack',
-        '-Wl,--no-warn-rwx-segments',
-]
-efi_ldflags += cc.get_supported_link_arguments(possible_link_flags)
+foreach arg : ['-Wl,--no-warn-execstack',
+               '-Wl,--no-warn-rwx-segments']
+        # We need to check the correct linker for supported args. This is what
+        # cc.has_multi_link_arguments() is for, but it helpfully overrides our
```

Comment:
Can you file an RFE for meson to support this natively? It looks like something generally useful I think?

Reply:
I doubt it would go trough. If I read the meson code correctly, this is because of msvc support.

/cc @eli-schwartz

Reply:
maybe an optional flag or something?